### ISSUE TITLE
Devtools locked up on the sources pane

### ISSUE THREAD

**Issue by 4ver** [nwjs/nw.js#2920]
I've been having an issue for the last few weeks where the devtools crashes when i go to the sources pane. When I close the crashed devtools window- the following error is output to the console.

```
[7901:0110/134053:ERROR:stream_listen_socket.cc(143)] send failed: errno==32
```

Now I have the problem where devtools always opens on the sources page so I have no way of using any of it. It's the same with the remote debugger.

Is there any way to reset the devtools cache? 

OSX 10.10 - NW 0.11.5

**Comment by adam-lynch** [comment]
:+1:. Only seen on a Mac too I think.

**Comment by ghostoy** [comment]
Could you try latest live-build to see if fixed your issues?
http://dl.node-webkit.org/live-build/01-14-2015/3f3500b-0962e6a-b477a2d-2e978ac-3e9fcbd-fd87c8d/

**Comment by 4ver** [comment]
Fixed as of 0.11.6
